tools/release/publish-release.ts

```diff
@@ -123,8 +123,12 @@ class PublishReleaseTask extends BaseReleaseTask {
     // the user to interactively confirm that the script should continue.
     await this._promptConfirmReleasePublish();
 
+    // Prompt for the OTP right before publishing so that it doesn't expire while building the
+    // packages.
+    const npmOtp = await promptForNpmOtp();
```

Comment:
Optional: just something I noticed after approving. Maybe we should have a comment here saying that the OTP token is needed as NPM requires two-factor auth for each write to the registry.